.papr.yml

```diff
@@ -12,6 +12,8 @@
 #   https://github.com/projectatomic/papr/blob/master/sample.papr.yml
 
 cluster:
+  specs:
+      ram: 4096
```

Comment:
Note the `specs` key is per host (e.g. you could have different specs for each host), so you need to repeat it three times for each item in the `hosts` list below. Might make sense to also accept a global cluster-level `specs` key as well.